package.json

```diff
@@ -10,11 +10,13 @@
   },
   "homepage": "https://github.com/Fitbit/fitbit-sdk-toolchain#readme",
   "scripts": {
-    "build": "rm -rf lib && tsc -p tsconfig.build.json",
+    "build": "rm -rf lib && npm run lint && tsc -p tsconfig.build.json",
     "test": "jest",
     "lint": "tsc --noEmit --noUnusedLocals && tslint --project .",
+    "lint-fix": "tsc --noEmit --noUnusedLocals && tslint --fix --project .",
     "depcheck": "depcheck --ignores='@types/*,tslint-config-airbnb' --ignore-dirs='toolchain,__test__'",
-    "prepublishOnly": "npm run test && npm run build"
+    "prepublishOnly": "npm run test && npm run build",
+    "precommit": "lint-staged && npm test"
```

Comment:
this could help automate the process a bit. This is only run when you commit your files. Why wouldn't you want to check the code on commit? Could be that you commit a change without realizing what it touches and could catch a bug early on.

You have veto, of course. Just let me know.